SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -16,6 +16,7 @@
 #include "LocalDeploymentManager.h"
 #include "LocalReceptionistProxyServerManager.h"
 #include "SpatialGDKEditorSettings.h"
+#include "SpatialPackageManager.h"
```

Comment:
Think this include can be moved to the `SpatialGDKEditorToolbar.cpp` since it's only used in there.